# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/pointerevents/pointerlock/pointerevent_getCoalescedEvents_when_pointerlocked.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/pointerevents/pointerlock/pointerevent_getCoalescedEvents_when_pointerlocked.html.ini
--- a/testing/web-platform/meta/pointerevents/pointerlock/pointerevent_getCoalescedEvents_when_pointerlocked.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/pointerevents/pointerlock/pointerevent_getCoalescedEvents_when_pointerlocked.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,13 +1,12 @@
 [pointerevent_getCoalescedEvents_when_pointerlocked.html]
   [mouse pointermove getCoalescedEvents when lock test]
     expected:
-      if (os == "win") and not fission and (processor == "x86_64") and not webrender and debug: ["FAIL", "PASS"]
-      if (os == "linux") and webrender and not swgl and debug and not fission: ["FAIL", "PASS"]
-      if (os == "win") and not fission and (processor == "x86") and debug: ["FAIL", "PASS"]
-      if (os == "linux") and webrender and not swgl and not debug: ["FAIL", "PASS"]
-      if (os == "linux") and webrender and swgl: ["FAIL", "PASS"]
+      if (os == "win") and debug and (processor == "x86_64") and webrender and not swgl: ["FAIL", "PASS"]
+      if (os == "win") and not debug and not fission and webrender: ["FAIL", "PASS"]
+      if (os == "win") and debug and (processor == "x86"): ["FAIL", "PASS"]
+      if (os == "linux") and fission and debug: ["FAIL", "PASS"]
       if (os == "android") and debug and webrender: ["FAIL", "PASS"]
       if (os == "android") and not debug and webrender: ["FAIL", "PASS"]
-      if (os == "android") and not debug and not webrender: ["FAIL", "PASS"]
+      if (os == "mac") and debug: ["FAIL", "PASS"]
       [PASS, FAIL]
 